src/AppInstallerCLITests/RestSourceTestHelper.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "TestSource.h"
+#include <AppInstallerRepositorySource.h>
+
+using namespace TestCommon;
+using namespace AppInstaller::Repository;
+
+// Test rest source.
+struct TestRestSource : public TestSource

```

Comment:
>TestRestSource [](http://example.com/codeflow?start=7&length=14)

nit: seems like not quite necessary to have this source specifically.

In all cases, you can
auto source = std::make_shared<tTestSource>:
source->Details = details;